src/main/java/com/bookstore/service/order/OrderItemServiceImpl.java

```diff
@@ -0,0 +1,37 @@
+package com.bookstore.service.order;
+
+import com.bookstore.dto.order.OrderItemDto;
+import com.bookstore.entity.Order;
+import com.bookstore.entity.OrderItem;
+import com.bookstore.exception.EntityNotFoundException;
+import com.bookstore.mapper.OrderMapper;
+import com.bookstore.repository.OrderItemRepository;
+import com.bookstore.repository.OrderRepository;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class OrderItemServiceImpl implements OrderItemService {
+    private final OrderRepository orderRepository;
+    private final OrderItemRepository orderItemRepository;
+    private final OrderMapper orderMapper;
+
+    @Override
+    public List<OrderItemDto> getOrderItems(Long orderId) {
+        Order order = orderRepository.findByIdWithOrderItems(orderId)
```

Comment:
replace to OrderService

```suggestion
        Order order = orderRepository.findByIdAndUserId(orderId, userId)
                .orElseThrow(()...
        return orderItemMapper.toOrderItemDtoList(order.getOrderItems());
```